src/isotools/_gene_plots.py

```diff
@@ -474,14 +493,16 @@ def gene_track(self, ax=None, title=None, reference=True, select_transcripts=Non
                 ax.text(pos, i + .25, enr, ha='center', va='center', color=contrast, fontsize=label_fontsize,
                         clip_on=True)  # bbox=dict(boxstyle='round', facecolor='wheat',edgecolor=None,  alpha=0.5)
         i += 1
-    if title is None:
-        title = f'{self.name} ({self.region})'
+    
+    if title is None: title = f'{self.name} ({self.region})'
```

Comment:
Why this change?
See [PEP 8 Style Guide](https://peps.python.org/pep-0008/#other-recommendations)
> Compound statements (multiple statements on the same line) are generally discouraged